users/views.py

```diff
@@ -1,3 +1,59 @@
-from django.shortcuts import render
+import json, re, bcrypt, jwt, requests
 
-# Create your views here.
+from django.shortcuts import render, redirect
+from django.views     import View
+from django.http      import JsonResponse
+from django.conf      import settings
+
+from users.models     import User
+
+class KakaoAPI:
+    def __init__(self, authorization_code):
+        self.authorization_code = authorization_code
+        self.token_url = "https://kauth.kakao.com/oauth/token"
+        self.user_url  = "https://kapi.kakao.com/v2/user/me"
+        self.redirect_url = "http://10.58.7.130:8000/users/kakao/callback"
+        
+    def get_access_token(self):
+        data = {
+            'grant_type': 'authorization_code',
+            'client_id': settings.KAKAO_APP_KEY,
+            'redirect_uri': self.redirect_url,
+            'code': self.authorization_code
+        }
+        response     = requests.post(self.token_url, data)
```

Comment:
`requests` 모듈을 사용하실 때에는 `timeout` 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 발생 합니다. timeout 옵션을 추가해 주세요.
```suggestion
        response     = requests.post(self.token_url, data, timeout = ..)
```

Reply:
보통 이렇게 third party 서버의 응답의 성공/실패 여부를 판단할때에는 status 코드로 비교해야 명확하고 디버깅하기 쉽습니다.
```python
response = requests.get(kakao_user_api_url, headers=headers)

if not response.status_code == 200:
    return JsonResponse...

            ..
```

Reply:
네 추가하겠습니다. status코드로 성공/실패 여부 판단도 추가하겠습니다.